bin/ping-server

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# from https://stackoverflow.com/questions/11904772/how-to-create-a-loop-in-bash-that-is-waiting-for-a-webserver-to-respond
+attempt_counter=0
+max_attempts=10
+
+until $(curl --output /dev/null --silent --head --fail http://localhost:3000); do
```

Comment:
I think this looks good, but wonder about timeouts, as [curl's docs say](https://everything.curl.dev/usingcurl/timeouts) "most operations in curl have no time-out by default!"

We might want to set a `--max-time` if we want to determine how long each iteration would last.

I also think that this could be done with the `--retry` param (which adds an increasing delay between retries), but that's not important.